README.md

```diff
@@ -21,16 +21,16 @@ With Project Flowgate, we can fetch metadata and runtime metrics from various fa
   - Open for other facility system integration.
 * **Built-in adapter for multiple IT stack systems**
   - vCenter Server
-  - vRealize Operation Manager
-  - Management Pack for vRealize Operation Manager Pack  ( Totally Free, [Download Here](https://marketplace.cloud.vmware.com/services/details/aaa?slug=true)) 
+  - Aria Operation Manager
+  - Management Pack for Aria Operation Manager Pack  ( Totally Free, [Download Here](https://marketplace.cloud.vmware.com/services/details/aaa?slug=true)) 
   - Open for other IT stack integration. More systems will coming soon.
 * **UI based Integration proccess**  One click integration. 
 * **Role based access control** API level access control support. 
 * **RestAPI support** Provide unified facility information querying services. APIs for all operations and data query make it easy to integrate with other systems.
 ## Get Start
 **DEMO**
 
-[Flowgate Management Pack for vROPs Demo](https://github.com/yixingjia/wormhole/releases/download/1.0/Flowgate_and_vROPs_Management_Pack_Demo.mp4)
+[Flowgate Management Pack for Aria Operations Demo](https://github.com/yixingjia/wormhole/releases/download/1.0/Flowgate_and_vROPs_Management_Pack_Demo.mp4)
```

Comment:
Let's remove Aria Operation.  Just leave "Flowgate Management Pack Demo"  ( since we don't update it for a long time)